src/content/contributing/translation-program/how-to-translate/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: How to translate
+lang: en
+description: Instructions for using Crowdin to translate ethereum.org
+sidebar: true
+---
+
+# How to translate {#how-to-translate}
```

Comment:
This page could use headings

<img width="853" alt="Screenshot 2022-08-08 at 17 41 23" src="https://user-images.githubusercontent.com/62268199/183471046-d8f27bae-0e01-4709-a882-feb570782e6c.png">

Reply:
Added headings here. Do you think that the 'Left sidebar', 'Online editor', and 'Right sidebar' sections should also have headings?

Reply:
LGTM. 

> Do you think that the 'Left sidebar', 'Online editor', and 'Right sidebar' sections should also have headings?

I wouldn't say so :-) 

The general rule I like to follow is: does the heading make sense independent of its content? If not, it's either not a heading, or the heading structure needs improving.

So, in this case: If we make these headings and look at the table of contents, we can see it doesn't really make sense.

```markdown
##Written guide
### Translate
#### Right sidebar
```